portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,83 @@
 // See the License for the specific language governing permissions and
```

Comment:
Can you pull all the tab specific login into its own file? Likewise for the slideshow logic.

Reply:
Will make a TODO to split up my script.js into logic for the slideshow and logic for tabs.